s3_upload/utils/utils.py

```
@@ -327,8 +327,19 @@ def get_sequencing_file_list(seq_dir, exclude_patterns=None) -> list:
+
+    Raises
+    ------
+    FileNotFoundError
+        Raised if the provided directory does not exist / not accessible
     """
     log.info("Getting list of files to upload in %s", seq_dir)
+
+    if not Path(seq_dir).absolute().exists():
+        raise FileNotFoundError(
+            f"Provided directory does not exist: {seq_dir}"
+        )
```

Ruff (0.8.0)

Avoid specifying long messages outside the exception class

(TRY003)